### ISSUE TITLE
improve accessibility with keyboard navigation and ARIA support

### ISSUE THREAD

**Issue by kubrickcode** [kubrickcode/specvital#313]
> Migrated from [specvital/web#190](https://github.com/specvital/web/issues/190)
> Original date: 2026-01-05

Screen reader and keyboard users were unable to navigate the spec view tree structure.
